src/Sentry.AspNetCore/SentryWebHostBuilderExtensions.cs

```diff
@@ -93,10 +93,16 @@ public static IWebHostBuilder UseSentry(
             _ = logging.Services
                 .AddSingleton<IConfigureOptions<SentryAspNetCoreOptions>, SentryAspNetCoreOptionsSetup>();
             _ = logging.Services.AddSingleton<ILoggerProvider, SentryAspNetCoreLoggerProvider>();
+            _ = logging.Services.AddSingleton<ILoggerProvider, SentryAspNetCoreStructuredLoggerProvider>();
 
             _ = logging.AddFilter<SentryAspNetCoreLoggerProvider>(
                 "Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware",
                 LogLevel.None);
+            _ = logging.AddFilter<SentryAspNetCoreStructuredLoggerProvider>(static (string? categoryName, LogLevel logLevel) =>
+            {
+                return categoryName is null
```

Comment:
are these guaranteed to be all of the categories we add?
I wonder if there's some "Constant" or something with these, we could reflect on and write a test to make sure we filter all of them out

Reply:
I'm fairly certain ... not quite positive though. Those are the `ILogger<T>` usages that we're using in our packages.
I definitely have to add a test for that ... probably as an ASP.NET Core integration test ... I see we have the infrastructure already setup via the "TestUtils" project.